packages/web-components/src/components/va-loading-indicator/va-loading-indicator.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Prop,
+  h,
+} from '@stencil/core';
+
+@Component({
+  tag: 'va-loading-indicator',
+  styleUrl: 'va-loading-indicator.css',
+  shadow: true,
+})
+export class VaLoadingIndicator {
+  @Element() el!: HTMLElement;
+  spinner!: HTMLDivElement;
+
+  loadingStartTime = null;
+  observer = null;
+
+  /**
+   * The message visible on screen when loading
+   */
+  @Prop() message: string;
+
+  /**
+   * An aria label
+   */
+  @Prop() label: string = 'Loading';
+
+  /**
+   * Set to true if the loading indicator should capture focus
+   */
+  @Prop() setFocus: boolean = false;
+
+  /**
+   * Analytics tracking function(s) will be called. Form components
+   * are disabled by default due to PII/PHI concerns.
```

Comment:
> ...due to PII/PHI concerns.

Well in this case, disabled because the component created millions of hits in GA no one cared about.  :grinning: